src/repositories/followers.repository.ts

```diff
@@ -0,0 +1,23 @@
+import { FollowerModel } from "../models/Follower.model";
+
+export const listFollowers = (queryParams: object) => {
+  const query = {};
+  const options = {
+    ...queryParams,
+  };
+  return FollowerModel.paginate(query, options);
+};
```

Comment:
```suggestion
export const listFollowers = (query: object, options: object) => {
   return FollowerModel.paginate(query, options);
};
```
check tweet repository